NuGet restore failing on .NET 5 for Debian-based OSes (CA trust changed)

# NuGet Team Update

For the most recent status and suggested workarounds please see: https://github.com/NuGet/Announcements/issues/49

# Original report from @dasMulli

Currently restoring any Microsoft packages within the official Docker images is broken and logging `NU3028` restore failures:

```Dockerfile
FROM mcr.microsoft.com/dotnet/sdk:5.0
WORKDIR /src
RUN dotnet new razor --auth Individual
RUN dotnet build
```

Output:
```sh
C:\demos\NuTest>docker build -t nutest .
Sending build context to Docker daemon  2.048kB
Step 1/4 : FROM mcr.microsoft.com/dotnet/sdk:5.0
 ---> e735190c5cb9
Step 2/4 : WORKDIR /src
 ---> Running in 05e3dacff59e
Removing intermediate container 05e3dacff59e
 ---> d102c9d21331
Step 3/4 : RUN dotnet new razor --auth Individual
 ---> Running in 6d1741e7e99b
Getting ready...
The template "ASP.NET Core Web App" was created successfully.
This template contains technologies from parties other than Microsoft, see https://aka.ms/aspnetcore/5.0-third-party-notices for details.

Processing post-creation actions...
Running 'dotnet restore' on /src/src.csproj...
  Determining projects to restore...
/src/src.csproj : error NU3028: Package 'System.Memory 4.5.3' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3037: Package 'System.Memory 4.5.3' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired./src/src.csproj : error NU3028: Package 'System.Memory 4.5.3' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.Logging.Abstractions 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3037: Package 'Microsoft.Extensions.Logging.Abstractions 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature 
validity period has expired.
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.Logging.Abstractions 5.0.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.Options 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3037: Package 'Microsoft.Extensions.Options 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired.
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.Options 5.0.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.Caching.Memory 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3037: Package 'Microsoft.Extensions.Caching.Memory 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired.
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.Caching.Memory 5.0.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3028: Package 'System.ComponentModel.Annotations 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3037: Package 'System.ComponentModel.Annotations 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired.
/src/src.csproj : error NU3028: Package 'System.ComponentModel.Annotations 5.0.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.DependencyInjection 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3037: Package 'Microsoft.Extensions.DependencyInjection 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired.
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.DependencyInjection 5.0.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3028: Package 'Microsoft.CSharp 4.7.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3037: Package 'Microsoft.CSharp 4.7.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired.
/src/src.csproj : error NU3028: Package 'Microsoft.CSharp 4.7.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.Configuration.Abstractions 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3037: Package 'Microsoft.Extensions.Configuration.Abstractions 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired.
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.Configuration.Abstractions 5.0.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.DependencyInjection.Abstractions 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3037: Package 'Microsoft.Extensions.DependencyInjection.Abstractions 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired.
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.DependencyInjection.Abstractions 5.0.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3028: Package 'Microsoft.DotNet.PlatformAbstractions 3.1.6' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3037: Package 'Microsoft.DotNet.PlatformAbstractions 3.1.6' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired.
/src/src.csproj : error NU3028: Package 'Microsoft.DotNet.PlatformAbstractions 3.1.6' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.Caching.Abstractions 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3037: Package 'Microsoft.Extensions.Caching.Abstractions 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature 
validity period has expired.
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.Caching.Abstractions 5.0.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3028: Package 'System.Collections.Immutable 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3037: Package 'System.Collections.Immutable 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired.
/src/src.csproj : error NU3028: Package 'System.Collections.Immutable 5.0.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.DependencyModel 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3037: Package 'Microsoft.Extensions.DependencyModel 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired.
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.DependencyModel 5.0.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's 
timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.FileProviders.Abstractions 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3037: Package 'Microsoft.Extensions.FileProviders.Abstractions 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired.
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.FileProviders.Abstractions 5.0.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.Logging 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3037: Package 'Microsoft.Extensions.Logging 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired.
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.Logging 5.0.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.Primitives 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3037: Package 'Microsoft.Extensions.Primitives 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired.
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.Primitives 5.0.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
  Failed to restore /src/src.csproj (in 3.24 sec).
Restore failed.
Post action failed.
Description: Restore NuGet packages required by this project.
Manual instructions: Run 'dotnet restore'

Removing intermediate container 6d1741e7e99b
 ---> ab6a0d574aeb
Step 4/4 : RUN dotnet build
 ---> Running in cf58e25578bf
Microsoft (R) Build Engine version 16.8.3+39993bd9d for .NET
Copyright (C) Microsoft Corporation. All rights reserved.

  Determining projects to restore...
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.DependencyInjection 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3037: Package 'Microsoft.Extensions.DependencyInjection 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired.
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.DependencyInjection 5.0.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.Primitives 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3037: Package 'Microsoft.Extensions.Primitives 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired.
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.Primitives 5.0.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.Configuration.Abstractions 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3037: Package 'Microsoft.Extensions.Configuration.Abstractions 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired.
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.Configuration.Abstractions 5.0.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.Logging 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3037: Package 'Microsoft.Extensions.Logging 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired.
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.Logging 5.0.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3028: Package 'System.ComponentModel.Annotations 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3037: Package 'System.ComponentModel.Annotations 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired.
/src/src.csproj : error NU3028: Package 'System.ComponentModel.Annotations 5.0.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.Caching.Abstractions 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3037: Package 'Microsoft.Extensions.Caching.Abstractions 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature 
validity period has expired.
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.Caching.Abstractions 5.0.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.FileProviders.Abstractions 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3037: Package 'Microsoft.Extensions.FileProviders.Abstractions 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired.
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.FileProviders.Abstractions 5.0.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3028: Package 'Microsoft.DotNet.PlatformAbstractions 3.1.6' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3037: Package 'Microsoft.DotNet.PlatformAbstractions 3.1.6' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired.
/src/src.csproj : error NU3028: Package 'Microsoft.DotNet.PlatformAbstractions 3.1.6' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.DependencyModel 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3037: Package 'Microsoft.Extensions.DependencyModel 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired.
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.DependencyModel 5.0.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's 
timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.Logging.Abstractions 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3037: Package 'Microsoft.Extensions.Logging.Abstractions 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature 
validity period has expired.
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.Logging.Abstractions 5.0.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.Caching.Memory 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3037: Package 'Microsoft.Extensions.Caching.Memory 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired.
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.Caching.Memory 5.0.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3028: Package 'System.Memory 4.5.3' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3037: Package 'System.Memory 4.5.3' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired./src/src.csproj : error NU3028: Package 'System.Memory 4.5.3' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3028: Package 'Microsoft.CSharp 4.7.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3037: Package 'Microsoft.CSharp 4.7.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired.
/src/src.csproj : error NU3028: Package 'Microsoft.CSharp 4.7.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3028: Package 'System.Collections.Immutable 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3037: Package 'System.Collections.Immutable 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired.
/src/src.csproj : error NU3028: Package 'System.Collections.Immutable 5.0.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.DependencyInjection.Abstractions 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3037: Package 'Microsoft.Extensions.DependencyInjection.Abstractions 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired.
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.DependencyInjection.Abstractions 5.0.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.Options 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3037: Package 'Microsoft.Extensions.Options 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired.
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.Options 5.0.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
  Failed to restore /src/src.csproj (in 846 ms).

Build FAILED.

/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.DependencyInjection 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3037: Package 'Microsoft.Extensions.DependencyInjection 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired.
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.DependencyInjection 5.0.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.Primitives 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3037: Package 'Microsoft.Extensions.Primitives 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired.
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.Primitives 5.0.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.Configuration.Abstractions 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3037: Package 'Microsoft.Extensions.Configuration.Abstractions 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired.
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.Configuration.Abstractions 5.0.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.Logging 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3037: Package 'Microsoft.Extensions.Logging 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired.
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.Logging 5.0.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3028: Package 'System.ComponentModel.Annotations 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3037: Package 'System.ComponentModel.Annotations 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired.
/src/src.csproj : error NU3028: Package 'System.ComponentModel.Annotations 5.0.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.Caching.Abstractions 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3037: Package 'Microsoft.Extensions.Caching.Abstractions 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature 
validity period has expired.
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.Caching.Abstractions 5.0.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.FileProviders.Abstractions 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3037: Package 'Microsoft.Extensions.FileProviders.Abstractions 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired.
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.FileProviders.Abstractions 5.0.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3028: Package 'Microsoft.DotNet.PlatformAbstractions 3.1.6' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3037: Package 'Microsoft.DotNet.PlatformAbstractions 3.1.6' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired.
/src/src.csproj : error NU3028: Package 'Microsoft.DotNet.PlatformAbstractions 3.1.6' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.DependencyModel 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3037: Package 'Microsoft.Extensions.DependencyModel 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired.
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.DependencyModel 5.0.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's 
timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.Logging.Abstractions 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3037: Package 'Microsoft.Extensions.Logging.Abstractions 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature 
validity period has expired.
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.Logging.Abstractions 5.0.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.Caching.Memory 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3037: Package 'Microsoft.Extensions.Caching.Memory 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired.
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.Caching.Memory 5.0.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3028: Package 'System.Memory 4.5.3' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3037: Package 'System.Memory 4.5.3' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired./src/src.csproj : error NU3028: Package 'System.Memory 4.5.3' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3028: Package 'Microsoft.CSharp 4.7.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3037: Package 'Microsoft.CSharp 4.7.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired.
/src/src.csproj : error NU3028: Package 'Microsoft.CSharp 4.7.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3028: Package 'System.Collections.Immutable 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3037: Package 'System.Collections.Immutable 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired.
/src/src.csproj : error NU3028: Package 'System.Collections.Immutable 5.0.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.DependencyInjection.Abstractions 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3037: Package 'Microsoft.Extensions.DependencyInjection.Abstractions 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired.
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.DependencyInjection.Abstractions 5.0.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.Options 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/src/src.csproj : error NU3037: Package 'Microsoft.Extensions.Options 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired.
/src/src.csproj : error NU3028: Package 'Microsoft.Extensions.Options 5.0.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
    0 Warning(s)
    48 Error(s)

Time Elapsed 00:00:01.28
The command '/bin/sh -c dotnet build' returned a non-zero code: 1
```

> Just put this repo online for you guys so you can see how it worked for me
>
> https://github.com/kiddieverts/nuget-fix

Hi, @kiddieverts!

I'm afraid this fix won't work. You haven't added a reference to `.csproj` into `.sln`, so running `dotnet restore` does nothing:

```
Step 6/12 : RUN dotnet restore
 ---> Running in d4a14952c115
  Determining projects to restore...
/usr/share/dotnet/sdk/5.0.102/NuGet.targets(131,5): warning : Unable to find a project to restore! [/app/dktest.sln]
```

Adding a project reference into an `.sln` file or running `dotnet restore ./dktest.csproj` produces an error:

```
Step 6/12 : RUN dotnet restore ./dktest.csproj
 ---> Running in a1952526eb55
  Determining projects to restore...
/app/dktest.csproj : error NU3028: Package 'System.Security.Cryptography.Cng 4.5.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain

... skipped for brevity ...

  Failed to restore /app/dktest.csproj (in 4.39 sec).
The command '/bin/sh -c dotnet restore ./dktest.csproj' returned a non-zero code: 1
```

It seems than `NoWarn="NU3028"` is ignored by `dotnet restore`.

> 
> 
>     Using the ubuntu image in the dockerfile is currently working for me (mcr.microsoft.com/dotnet/sdk:5.0-focal)
> 
> This is probably the most useful answer so far, requires no changes other than swapping the sdk builder container to ubuntu based

Still, this does not work for all packages.

```
/app/baam.csproj : error NU3034: Package 'runtime.win-x64.runtime.native.System.Data.SqlClient.sni 4.4.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer.
/app/baam.csproj : error NU3034: Package 'runtime.win-x86.runtime.native.System.Data.SqlClient.sni 4.4.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer.
/app/baam.csproj : error NU3034: Package 'runtime.win-arm64.runtime.native.System.Data.SqlClient.sni 4.4.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer.
/app/baam.csproj : error NU3034: Package 'MSBuildGitHash 2.0.2' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer.
/app/baam.csproj : error NU3034: Package 'Swashbuckle.AspNetCore 5.6.3' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer.
/app/baam.csproj : error NU3034: Package 'Packaging.Targets 0.1.171' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer.
/app/baam.csproj : error NU3034: Package 'Swashbuckle.AspNetCore.Swagger 5.6.3' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer.
/app/baam.csproj : error NU3034: Package 'Swashbuckle.AspNetCore.SwaggerGen 5.6.3' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer.
/app/baam.csproj : error NU3034: Package 'Swashbuckle.AspNetCore.SwaggerUI 5.6.3' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer.
/app/baam.csproj : error NU3034: Package 'Newtonsoft.Json 12.0.2' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer.
/app/baam.csproj : error NU3034: Package 'Newtonsoft.Json.Bson 1.0.2' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer.
/app/baam.csproj : error NU3034: Package 'Humanizer.Core 2.8.26' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer.
/app/baam.csproj : error NU3034: Package 'SQLitePCLRaw.bundle_e_sqlite3 2.0.4' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer.
/app/baam.csproj : error NU3034: Package 'SQLitePCLRaw.core 2.0.4' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer.
/app/baam.csproj : error NU3034: Package 'SQLitePCLRaw.provider.dynamic_cdecl 2.0.4' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer.
/app/baam.csproj : error NU3034: Package 'SQLitePCLRaw.lib.e_sqlite3 2.0.4' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer.
```

The `5.0-focal` sdk works at least for my production scenario and unblocks me for now.

For my initial repo starting this issue above it reduces the number of failed packages but some still fail:
```Dockerfile
FROM mcr.microsoft.com/dotnet/sdk:5.0-focal
WORKDIR /src
RUN echo '<?xml version="1.0" encoding="utf-8"?>\
<configuration>\
  <packageSources>\
    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />\
  </packageSources>\
\
  <config>\
    <add key="signatureValidationMode" value="require" />\
  </config>\
\
  <trustedSigners>\
    <author name="Microsoft">\
      <certificate fingerprint="3F9001EA83C560D712C24CF213C3D312CB3BFF51EE89435D3430BD06B5D0EECE" hashAlgorithm="SHA256" allowUntrustedRoot="false" />\
      <certificate fingerprint="AA12DA22A49BCE7D5C1AE64CC1F3D892F150DA76140F210ABD2CBFFCA2C18A27" hashAlgorithm="SHA256" allowUntrustedRoot="false" />\
    </author>\
  </trustedSigners>\
\
</configuration>' >> NuGet.Config
RUN dotnet new razor --auth Individual
RUN dotnet build
```

```
Step 5/5 : RUN dotnet build
 ---> Running in ce01b71d06db
Microsoft (R) Build Engine version 16.8.3+39993bd9d for .NET
Copyright (C) Microsoft Corporation. All rights reserved.

  Determining projects to restore...
/src/src.csproj : error NU3034: Package 'SQLitePCLRaw.lib.e_sqlite3 2.0.4' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer.
/src/src.csproj : error NU3034: Package 'SQLitePCLRaw.core 2.0.4' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer.
/src/src.csproj : error NU3034: Package 'SQLitePCLRaw.bundle_e_sqlite3 2.0.4' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer.
/src/src.csproj : error NU3034: Package 'SQLitePCLRaw.provider.dynamic_cdecl 2.0.4' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by 
a trusted signer.
/src/src.csproj : error NU3034: Package 'Humanizer.Core 2.8.26' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer.   
  Failed to restore /src/src.csproj (in 838 ms).

Build FAILED.
```

> The `5.0-focal` sdk works at least for my production scenario and unblocks me for now.
> 
> For my initial repo starting this issue above it reduces the number of failed packages but some still fail:
> 
> ```dockerfile
> FROM mcr.microsoft.com/dotnet/sdk:5.0-focal
> WORKDIR /src
> RUN echo '<?xml version="1.0" encoding="utf-8"?>\
> <configuration>\
>   <packageSources>\
>     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />\
>   </packageSources>\
> \
>   <config>\
>     <add key="signatureValidationMode" value="require" />\
>   </config>\
> \
>   <trustedSigners>\
>     <author name="Microsoft">\
>       <certificate fingerprint="3F9001EA83C560D712C24CF213C3D312CB3BFF51EE89435D3430BD06B5D0EECE" hashAlgorithm="SHA256" allowUntrustedRoot="false" />\
>       <certificate fingerprint="AA12DA22A49BCE7D5C1AE64CC1F3D892F150DA76140F210ABD2CBFFCA2C18A27" hashAlgorithm="SHA256" allowUntrustedRoot="false" />\
>     </author>\
>   </trustedSigners>\
> \
> </configuration>' >> NuGet.Config
> RUN dotnet new razor --auth Individual
> RUN dotnet build
> ```
> 
> ```
> Step 5/5 : RUN dotnet build
>  ---> Running in ce01b71d06db
> Microsoft (R) Build Engine version 16.8.3+39993bd9d for .NET
> Copyright (C) Microsoft Corporation. All rights reserved.
> 
>   Determining projects to restore...
> /src/src.csproj : error NU3034: Package 'SQLitePCLRaw.lib.e_sqlite3 2.0.4' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer.
> /src/src.csproj : error NU3034: Package 'SQLitePCLRaw.core 2.0.4' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer.
> /src/src.csproj : error NU3034: Package 'SQLitePCLRaw.bundle_e_sqlite3 2.0.4' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer.
> /src/src.csproj : error NU3034: Package 'SQLitePCLRaw.provider.dynamic_cdecl 2.0.4' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by 
> a trusted signer.
> /src/src.csproj : error NU3034: Package 'Humanizer.Core 2.8.26' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer.   
>   Failed to restore /src/src.csproj (in 838 ms).
> 
> Build FAILED.
> ```

Remove the trusted signer bit you are pasting into the nuget.config and it will build

> > The `5.0-focal` sdk works at least for my production scenario and unblocks me for now.
> > For my initial repo starting this issue above it reduces the number of failed packages but some still fail:
> > ```dockerfile
> > FROM mcr.microsoft.com/dotnet/sdk:5.0-focal
> > WORKDIR /src
> > RUN echo '<?xml version="1.0" encoding="utf-8"?>\
> > <configuration>\
> >   <packageSources>\
> >     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />\
> >   </packageSources>\
> > \
> >   <config>\
> >     <add key="signatureValidationMode" value="require" />\
> >   </config>\
> > \
> >   <trustedSigners>\
> >     <author name="Microsoft">\
> >       <certificate fingerprint="3F9001EA83C560D712C24CF213C3D312CB3BFF51EE89435D3430BD06B5D0EECE" hashAlgorithm="SHA256" allowUntrustedRoot="false" />\
> >       <certificate fingerprint="AA12DA22A49BCE7D5C1AE64CC1F3D892F150DA76140F210ABD2CBFFCA2C18A27" hashAlgorithm="SHA256" allowUntrustedRoot="false" />\
> >     </author>\
> >   </trustedSigners>\
> > \
> > </configuration>' >> NuGet.Config
> > RUN dotnet new razor --auth Individual
> > RUN dotnet build
> > ```
> > 
> > 
> > ```
> > Step 5/5 : RUN dotnet build
> >  ---> Running in ce01b71d06db
> > Microsoft (R) Build Engine version 16.8.3+39993bd9d for .NET
> > Copyright (C) Microsoft Corporation. All rights reserved.
> > 
> >   Determining projects to restore...
> > /src/src.csproj : error NU3034: Package 'SQLitePCLRaw.lib.e_sqlite3 2.0.4' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer.
> > /src/src.csproj : error NU3034: Package 'SQLitePCLRaw.core 2.0.4' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer.
> > /src/src.csproj : error NU3034: Package 'SQLitePCLRaw.bundle_e_sqlite3 2.0.4' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer.
> > /src/src.csproj : error NU3034: Package 'SQLitePCLRaw.provider.dynamic_cdecl 2.0.4' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by 
> > a trusted signer.
> > /src/src.csproj : error NU3034: Package 'Humanizer.Core 2.8.26' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer.   
> >   Failed to restore /src/src.csproj (in 838 ms).
> > 
> > Build FAILED.
> > ```
> 
> Remove the trusted signer bit you are pasting into the nuget.config and it will build

Thanks it worked

Our preliminary investigation shows that .NET 5 restore is broken on some operating systems resulting in error messages like:
 
error NU3037: Package 'System.Memory 4.5.3' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired.
 
Known workarounds include:
1. Downgrade to .NET Core 3.1
2. If using docker, change your base image from "FROM mcr.microsoft.com/dotnet/sdk:5.0" to "
FROM mcr.microsoft.com/dotnet/sdk:5.0-focal" or "
FROM mcr.microsoft.com/dotnet/sdk:5.0-alpine"

This was triggered by the expiration of the previous Microsoft signing certificate as was described here: https://devblogs.microsoft.com/nuget/microsoft-author-signing-certificate-update/ 

that caused the issue on some OSS.

We are continuing investigation and will let you know when learn more!

> Our preliminary investigation shows that .NET 5 restore is broken on some operating systems resulting in error messages like:
> 
> error NU3037: Package 'System.Memory 4.5.3' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired.
> 
> Known workarounds include:
> 
> 1. Downgrade to .NET Core 3.1
> 2. If using docker, change your base image from "FROM mcr.microsoft.com/dotnet/sdk:5.0" to "
>    FROM mcr.microsoft.com/dotnet/sdk:5.0-focal" or "
>    FROM mcr.microsoft.com/dotnet/sdk:5.0-alpine"
> 
> This was triggered by the expiration of the previous Microsoft signing certificate as was described here: https://devblogs.microsoft.com/nuget/microsoft-author-signing-certificate-update/
> 
> that caused the issue on some OSS.
> 
> We are continuing investigation and will let you know when learn more!

"Some OSS". Understatement of the year :sweat_smile:

Moving from buster-slim to alpine doesn't work for me somehow :(.

```
.Web.csproj : error NU3028: Package 'Microsoft.CSharp 4.7.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain [...]
```

error NU3028: Package 'Microsoft.Win32.SystemEvents 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain

> @EnthusedDragon, could you also share the broken restore build output you were seeing?

Sure, hope this helps.



```
2021-01-28T07:44:03.1604628Z ##[section]Starting: Dotnet Test
2021-01-28T07:44:03.1613666Z ==============================================================================
2021-01-28T07:44:03.1614004Z Task         : .NET Core
2021-01-28T07:44:03.1614337Z Description  : Build, test, package, or publish a dotnet application, or run a custom dotnet command
2021-01-28T07:44:03.1614686Z Version      : 2.181.0
2021-01-28T07:44:03.1614947Z Author       : Microsoft Corporation
2021-01-28T07:44:03.1615307Z Help         : https://docs.microsoft.com/azure/devops/pipelines/tasks/build/dotnet-core-cli
2021-01-28T07:44:03.1615795Z ==============================================================================
2021-01-28T07:44:03.5107956Z Info: .NET Core SDK/runtime 2.2 and 3.0 are now End of Life(EOL) and have been removed from all hosted agents. If you're using these SDK/runtimes on hosted agents, kindly upgrade to newer versions which are not EOL, or else use UseDotNet task to install the required version.
2021-01-28T07:44:03.5174206Z [command]/usr/bin/dotnet test /workspace/100/s/MyProject.V2.Notification.sln --logger trx --results-directory /workspace/_temp --configuration Release /p:CollectCoverage=true /p:CoverletOutputFormat=cobertura /p:thresholdType=line /p:thresholdStat=total
2021-01-28T07:44:05.1200137Z   Determining projects to restore...
2021-01-28T07:44:07.4678312Z   Restored /workspace/100/s/Tests/MyProject.V2.Providers.ITouch.Tests/MyProject.V2.Providers.ITouch.Tests.csproj (in 1.56 sec).
2021-01-28T07:44:07.4679114Z   Restored /workspace/100/s/Tests/MyProject.V2.Providers.OtherCompany.Tests/MyProject.V2.Providers.OtherCompany.Tests.csproj (in 1.56 sec).
2021-01-28T07:44:07.9592778Z   Restored /workspace/100/s/Tests/MyProject.V2.Providers.Company.Tests/MyProject.V2.Providers.Company.Tests.csproj (in 480 ms).
2021-01-28T07:44:11.4991338Z /workspace/100/s/Tests/MyProject.V2.Notification.Worker.Tests/MyProject.V2.Notification.Worker.Tests.csproj : error NU3028: Package 'Microsoft.CodeAnalysis.Common 3.6.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain [/workspace/100/s/MyProject.V2.Notification.sln]
2021-01-28T07:44:11.4994145Z /workspace/100/s/Tests/MyProject.V2.Notification.Worker.Tests/MyProject.V2.Notification.Worker.Tests.csproj : error NU3037: Package 'Microsoft.CodeAnalysis.Common 3.6.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired. [/workspace/100/s/MyProject.V2.Notification.sln]
2021-01-28T07:44:11.4996463Z /workspace/100/s/Tests/MyProject.V2.Notification.Worker.Tests/MyProject.V2.Notification.Worker.Tests.csproj : error NU3028: Package 'Microsoft.CodeAnalysis.Common 3.6.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain [/workspace/100/s/MyProject.V2.Notification.sln]
2021-01-28T07:44:11.4998678Z /workspace/100/s/Tests/MyProject.V2.Notification.Worker.Tests/MyProject.V2.Notification.Worker.Tests.csproj : error NU3028: Package 'Microsoft.CodeAnalysis.Analyzers 3.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain [/workspace/100/s/MyProject.V2.Notification.sln]
2021-01-28T07:44:11.5000912Z /workspace/100/s/Tests/MyProject.V2.Notification.Worker.Tests/MyProject.V2.Notification.Worker.Tests.csproj : error NU3037: Package 'Microsoft.CodeAnalysis.Analyzers 3.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired. [/workspace/100/s/MyProject.V2.Notification.sln]
2021-01-28T07:44:11.5002978Z /workspace/100/s/Tests/MyProject.V2.Notification.Worker.Tests/MyProject.V2.Notification.Worker.Tests.csproj : error NU3028: Package 'Microsoft.CodeAnalysis.Analyzers 3.0.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain [/workspace/100/s/MyProject.V2.Notification.sln]
2021-01-28T07:44:11.5123821Z /workspace/100/s/Tests/MyProject.V2.Notification.Worker.Tests/MyProject.V2.Notification.Worker.Tests.csproj : warning NU3037: Package 'Microsoft.CodeAnalysis.CSharp 3.6.0' from source 'https://nexus-agl.company.co.za/repository/nuget.org-proxy/': The author primary signature validity period has expired. [/workspace/100/s/MyProject.V2.Notification.sln]
2021-01-28T07:44:11.7850667Z   Restored /workspace/100/s/Tests/MyProject.V2.Providers.Aws.Tests/MyProject.V2.Providers.Aws.Tests.csproj (in 4.29 sec).
2021-01-28T07:44:11.9630762Z /workspace/100/s/Tests/MyProject.V2.Notification.Domain.Tests/MyProject.V2.Notification.Domain.Tests.csproj : error NU3028: Package 'Microsoft.CodeAnalysis.Analyzers 3.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain [/workspace/100/s/MyProject.V2.Notification.sln]
2021-01-28T07:44:11.9673780Z /workspace/100/s/Tests/MyProject.V2.Notification.Domain.Tests/MyProject.V2.Notification.Domain.Tests.csproj : error NU3037: Package 'Microsoft.CodeAnalysis.Analyzers 3.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired. [/workspace/100/s/MyProject.V2.Notification.sln]
2021-01-28T07:44:11.9676893Z /workspace/100/s/Tests/MyProject.V2.Notification.Domain.Tests/MyProject.V2.Notification.Domain.Tests.csproj : error NU3028: Package 'Microsoft.CodeAnalysis.Analyzers 3.0.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain [/workspace/100/s/MyProject.V2.Notification.sln]
2021-01-28T07:44:11.9840119Z /workspace/100/s/Tests/MyProject.V2.Notification.Domain.Tests/MyProject.V2.Notification.Domain.Tests.csproj : error NU3028: Package 'Microsoft.CodeAnalysis.Common 3.6.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain [/workspace/100/s/MyProject.V2.Notification.sln]
2021-01-28T07:44:11.9842129Z /workspace/100/s/Tests/MyProject.V2.Notification.Domain.Tests/MyProject.V2.Notification.Domain.Tests.csproj : error NU3037: Package 'Microsoft.CodeAnalysis.Common 3.6.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired. [/workspace/100/s/MyProject.V2.Notification.sln]
2021-01-28T07:44:11.9843995Z /workspace/100/s/Tests/MyProject.V2.Notification.Domain.Tests/MyProject.V2.Notification.Domain.Tests.csproj : error NU3028: Package 'Microsoft.CodeAnalysis.Common 3.6.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain [/workspace/100/s/MyProject.V2.Notification.sln]
2021-01-28T07:44:12.0278599Z /workspace/100/s/Tests/MyProject.V2.Notification.Domain.Tests/MyProject.V2.Notification.Domain.Tests.csproj : warning NU3037: Package 'Microsoft.CodeAnalysis.CSharp 3.6.0' from source 'https://nexus-agl.company.co.za/service/local/nuget/cib-nuget-group/': The author primary signature validity period has expired. [/workspace/100/s/MyProject.V2.Notification.sln]
2021-01-28T07:44:12.1710573Z   Failed to restore /workspace/100/s/Tests/MyProject.V2.Notification.Domain.Tests/MyProject.V2.Notification.Domain.Tests.csproj (in 332 ms).
2021-01-28T07:44:12.1711262Z   Failed to restore /workspace/100/s/Tests/MyProject.V2.Notification.Worker.Tests/MyProject.V2.Notification.Worker.Tests.csproj (in 4.17 sec).
2021-01-28T07:44:12.2273600Z   Restored /workspace/100/s/Tests/MyProject.V2.Notification.Dal.PostgreSql.Tests/MyProject.V2.Notification.Dal.PostgreSql.Tests.csproj (in 95 ms).
2021-01-28T07:44:12.2316724Z   Restored /workspace/100/s/Tests/MyProject.V2.Notification.Common.Tests/MyProject.V2.Notification.Common.Tests.csproj (in 55 ms).
2021-01-28T07:44:14.5009232Z /workspace/100/s/Tests/MyProject.V2.Notification.Api.Tests/MyProject.V2.Notification.Api.Tests.csproj : warning NU3037: Package 'System.Reflection.Metadata 5.0.0' from source 'https://nexus-agl.company.co.za/repository/nuget.org-proxy/': The author primary signature validity period has expired. [/workspace/100/s/MyProject.V2.Notification.sln]
2021-01-28T07:44:14.9694575Z /workspace/100/s/Tests/MyProject.V2.Notification.Api.Tests/MyProject.V2.Notification.Api.Tests.csproj : warning NU3037: Package 'Microsoft.CodeAnalysis.CSharp.Workspaces 3.8.0' from source 'https://nexus-agl.company.co.za/service/local/nuget/cib-nuget-group/': The author primary signature validity period has expired. [/workspace/100/s/MyProject.V2.Notification.sln]
2021-01-28T07:44:14.9705835Z /workspace/100/s/Tests/MyProject.V2.Notification.Api.Tests/MyProject.V2.Notification.Api.Tests.csproj : error NU3028: Package 'Microsoft.CodeAnalysis.Common 3.6.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain [/workspace/100/s/MyProject.V2.Notification.sln]
2021-01-28T07:44:14.9709929Z /workspace/100/s/Tests/MyProject.V2.Notification.Api.Tests/MyProject.V2.Notification.Api.Tests.csproj : error NU3037: Package 'Microsoft.CodeAnalysis.Common 3.6.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired. [/workspace/100/s/MyProject.V2.Notification.sln]
2021-01-28T07:44:14.9712060Z /workspace/100/s/Tests/MyProject.V2.Notification.Api.Tests/MyProject.V2.Notification.Api.Tests.csproj : error NU3028: Package 'Microsoft.CodeAnalysis.Common 3.6.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain [/workspace/100/s/MyProject.V2.Notification.sln]
2021-01-28T07:44:14.9714180Z /workspace/100/s/Tests/MyProject.V2.Notification.Api.Tests/MyProject.V2.Notification.Api.Tests.csproj : warning NU3037: Package 'Microsoft.CodeAnalysis.Analyzers 3.0.0' from source 'https://nexus-agl.company.co.za/service/local/nuget/cib-nuget-group/': The author primary signature validity period has expired. [/workspace/100/s/MyProject.V2.Notification.sln]
2021-01-28T07:44:14.9716155Z /workspace/100/s/Tests/MyProject.V2.Notification.Api.Tests/MyProject.V2.Notification.Api.Tests.csproj : warning NU3037: Package 'Microsoft.CodeAnalysis.CSharp 3.6.0' from source 'https://nexus-agl.company.co.za/service/local/nuget/cib-nuget-group/': The author primary signature validity period has expired. [/workspace/100/s/MyProject.V2.Notification.sln]
2021-01-28T07:44:14.9718220Z /workspace/100/s/Tests/MyProject.V2.Notification.Api.Tests/MyProject.V2.Notification.Api.Tests.csproj : error NU3028: Package 'Microsoft.CodeAnalysis.Workspaces.Common 3.8.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain [/workspace/100/s/MyProject.V2.Notification.sln]
2021-01-28T07:44:14.9726208Z /workspace/100/s/Tests/MyProject.V2.Notification.Api.Tests/MyProject.V2.Notification.Api.Tests.csproj : error NU3037: Package 'Microsoft.CodeAnalysis.Workspaces.Common 3.8.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired. [/workspace/100/s/MyProject.V2.Notification.sln]
2021-01-28T07:44:14.9728644Z /workspace/100/s/Tests/MyProject.V2.Notification.Api.Tests/MyProject.V2.Notification.Api.Tests.csproj : error NU3028: Package 'Microsoft.CodeAnalysis.Workspaces.Common 3.8.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain [/workspace/100/s/MyProject.V2.Notification.sln]
2021-01-28T07:44:14.9730050Z   Restored /workspace/100/s/Tests/MyProject.V2.Notification.BFF.Tests/MyProject.V2.Notification.BFF.Tests.csproj (in 2.55 sec).
2021-01-28T07:44:14.9733426Z /workspace/100/s/Tests/MyProject.V2.Notification.Api.Tests/MyProject.V2.Notification.Api.Tests.csproj : error NU3028: Package 'Microsoft.CodeAnalysis.Common 3.8.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain [/workspace/100/s/MyProject.V2.Notification.sln]
2021-01-28T07:44:14.9735316Z /workspace/100/s/Tests/MyProject.V2.Notification.Api.Tests/MyProject.V2.Notification.Api.Tests.csproj : error NU3037: Package 'Microsoft.CodeAnalysis.Common 3.8.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired. [/workspace/100/s/MyProject.V2.Notification.sln]
2021-01-28T07:44:14.9737166Z /workspace/100/s/Tests/MyProject.V2.Notification.Api.Tests/MyProject.V2.Notification.Api.Tests.csproj : error NU3028: Package 'Microsoft.CodeAnalysis.Common 3.8.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain [/workspace/100/s/MyProject.V2.Notification.sln]
2021-01-28T07:44:14.9739056Z /workspace/100/s/Tests/MyProject.V2.Notification.Api.Tests/MyProject.V2.Notification.Api.Tests.csproj : warning NU3037: Package 'Microsoft.CodeAnalysis.CSharp 3.8.0' from source 'https://nexus-agl.company.co.za/service/local/nuget/cib-nuget-group/': The author primary signature validity period has expired. [/workspace/100/s/MyProject.V2.Notification.sln]
2021-01-28T07:44:15.8020934Z /workspace/100/s/Tests/MyProject.V2.Notification.Api.Integration.Tests/MyProject.V2.Notification.Api.Integration.Tests.csproj : warning NU3037: Package 'Microsoft.CodeAnalysis.Analyzers 3.0.0' from source 'https://nexus-agl.company.co.za/service/local/nuget/cib-nuget-group/': The author primary signature validity period has expired. [/workspace/100/s/MyProject.V2.Notification.sln]
2021-01-28T07:44:15.8025555Z /workspace/100/s/Tests/MyProject.V2.Notification.Api.Integration.Tests/MyProject.V2.Notification.Api.Integration.Tests.csproj : warning NU3037: Package 'Microsoft.CodeAnalysis.Common 3.6.0' from source 'https://nexus-agl.company.co.za/service/local/nuget/cib-nuget-group/': The author primary signature validity period has expired. [/workspace/100/s/MyProject.V2.Notification.sln]
2021-01-28T07:44:15.8028338Z /workspace/100/s/Tests/MyProject.V2.Notification.Api.Integration.Tests/MyProject.V2.Notification.Api.Integration.Tests.csproj : warning NU3037: Package 'Microsoft.CodeAnalysis.Common 3.8.0' from source 'https://nexus-agl.company.co.za/service/local/nuget/cib-nuget-group/': The author primary signature validity period has expired. [/workspace/100/s/MyProject.V2.Notification.sln]
2021-01-28T07:44:15.8030744Z /workspace/100/s/Tests/MyProject.V2.Notification.Api.Integration.Tests/MyProject.V2.Notification.Api.Integration.Tests.csproj : warning NU3037: Package 'Microsoft.CodeAnalysis.Workspaces.Common 3.8.0' from source 'https://nexus-agl.company.co.za/service/local/nuget/cib-nuget-group/': The author primary signature validity period has expired. [/workspace/100/s/MyProject.V2.Notification.sln]
2021-01-28T07:44:15.8032350Z   Failed to restore /workspace/100/s/Tests/MyProject.V2.Notification.Api.Tests/MyProject.V2.Notification.Api.Tests.csproj (in 2.98 sec).
2021-01-28T07:44:15.8033079Z   Restored /workspace/100/s/Source/MyProject.V2.Providers.Shugo/MyProject.V2.Providers.Shugo.csproj (in 11 ms).
2021-01-28T07:44:15.8033714Z   Restored /workspace/100/s/Tests/MyProject.V2.Providers.Shugo.Tests/MyProject.V2.Providers.Shugo.Tests.csproj (in 46 ms).
2021-01-28T07:44:15.8034314Z   Restored /workspace/100/s/Source/MyProject.V2.Providers.ITouch/MyProject.V2.Providers.ITouch.csproj (in 4 ms).
2021-01-28T07:44:15.8161593Z   Restored /workspace/100/s/Source/MyProject.V2.Providers.OtherCompany/MyProject.V2.Providers.OtherCompany.csproj (in 10 ms).
2021-01-28T07:44:15.8741410Z   Restored /workspace/100/s/Source/MyProject.V2.Providers.Aws/MyProject.V2.Providers.Aws.csproj (in 14 ms).
2021-01-28T07:44:15.8950402Z   Restored /workspace/100/s/Source/MyProject.V2.Providers.Company/MyProject.V2.Providers.Company.csproj (in 18 ms).
2021-01-28T07:44:16.6616110Z /workspace/100/s/Tests/MyProject.V2.Notification.Api.Integration.Tests/MyProject.V2.Notification.Api.Integration.Tests.csproj : warning NU3037: Package 'Microsoft.CodeAnalysis.CSharp 3.6.0' from source 'https://nexus-agl.company.co.za/service/local/nuget/cib-nuget-group/': The author primary signature validity period has expired. [/workspace/100/s/MyProject.V2.Notification.sln]
2021-01-28T07:44:16.6618265Z /workspace/100/s/Tests/MyProject.V2.Notification.Api.Integration.Tests/MyProject.V2.Notification.Api.Integration.Tests.csproj : warning NU3037: Package 'Microsoft.CodeAnalysis.CSharp 3.8.0' from source 'https://nexus-agl.company.co.za/service/local/nuget/cib-nuget-group/': The author primary signature validity period has expired. [/workspace/100/s/MyProject.V2.Notification.sln]
2021-01-28T07:44:16.6619264Z   Restored /workspace/100/s/Source/MyProject.V2.Notification.Worker/MyProject.V2.Notification.Worker.csproj (in 675 ms).
2021-01-28T07:44:16.6619874Z   Restored /workspace/100/s/Tests/MyProject.V2.Notification.Api.Integration.Tests/MyProject.V2.Notification.Api.Integration.Tests.csproj (in 1.8 sec).
2021-01-28T07:44:16.6620889Z   Restored /workspace/100/s/Source/MyProject.V2.Notification.Domain/MyProject.V2.Notification.Domain.csproj (in 24 ms).
2021-01-28T07:44:16.6621878Z   Restored /workspace/100/s/Source/MyProject.V2.Notification.Dal.PostgreSql/MyProject.V2.Notification.Dal.PostgreSql.csproj (in 17 ms).
2021-01-28T07:44:16.6622674Z   Restored /workspace/100/s/Source/MyProject.V2.Notification.Common/MyProject.V2.Notification.Common.csproj (in 7 ms).
2021-01-28T07:44:16.6757796Z   Restored /workspace/100/s/Source/MyProject.V2.Notification.Dal.PostgreSql.Migrator/MyProject.V2.Notification.Dal.PostgreSql.Migrator.csproj (in 26 ms).
2021-01-28T07:44:16.7788987Z   Restored /workspace/100/s/Source/MyProject.V2.Notification.BFF/MyProject.V2.Notification.BFF.csproj (in 92 ms).
2021-01-28T07:44:16.7908420Z   Restored /workspace/100/s/Source/MyProject.V2.Notification.Abstractions/MyProject.V2.Notification.Abstractions.csproj (in 11 ms).
2021-01-28T07:44:16.7926543Z   Restored /workspace/100/s/Source/MyProject.V2.Notification.Api/MyProject.V2.Notification.Api.csproj (in 107 ms).
2021-01-28T07:44:16.8580753Z ##[error]Error: The process '/usr/bin/dotnet' failed with exit code 1
2021-01-28T07:44:16.8599031Z ##[warning]No test result files were found.
2021-01-28T07:44:16.8602044Z ##[warning].NET 5 has some compatibility issues with older Nuget versions(<=5.7), so if you are using an older Nuget version(and not dotnet cli) to restore, then the dotnet cli commands (e.g. dotnet build) which rely on such restored packages might fail. To mitigate such error, you can either: (1) - Use dotnet cli to restore, (2) - Use Nuget version 5.8 to restore, (3) - Use global.json using an older sdk version(<=3) to build
2021-01-28T07:44:16.8605320Z Info: Azure Pipelines hosted agents have been updated and now contain .Net 5.x SDK/Runtime along with the older .Net Core version which are currently lts. Unless you have locked down a SDK version for your project(s), 5.x SDK might be picked up which might have breaking behavior as compared to previous versions. You can learn more about the breaking changes here: https://docs.microsoft.com/en-us/dotnet/core/tools/ and https://docs.microsoft.com/en-us/dotnet/core/compatibility/ . To learn about more such changes and troubleshoot, refer here: https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/build/dotnet-core-cli?view=azure-devops#troubleshooting
2021-01-28T07:44:16.8607928Z ##[error]Dotnet command failed with non-zero exit code on the following projects : /workspace/100/s/MyProject.V2.Notification.sln
2021-01-28T07:44:16.8633800Z ##[section]Finishing: Dotnet Test
```

for me, even using 5.0.102-ca-patch-buster-slim image tag did not resolve the issues, i have less error, but it persist for other dependencies such as 

 error NU3034: Package 'StackExchange.Redis 2.2.4' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer.
 error NU3034: Package 'System.Net.NameResolution 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer.
 error NU3034: Package 'Pipelines.Sockets.Unofficial 2.2.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer.

@JonDouglas Not fully convinced that this is entirely solved yet. I am seeing issue on Debian Bullseye, trying to build this commit in my project: https://github.com/perlang-org/perlang/commit/6a0802af511b43b4143e045d6dfeb9130d88d9c4 (but I just checked and it has started failing on the latest master as well; this is utterly strange since I'm 100% positive the last `master` commit _worked_ yesterday. I have cleared my `~/.nuget` folder when debugging this so it may have made things worse somehow.)

`dotnet build` on this commit (which is a .NET Core 3.1-based tree) fails with errors like below. As noted, the latest `master` (which uses .NET 5) fails likewise.

```
Microsoft (R) Build Engine version 16.8.3+39993bd9d for .NET
Copyright (C) Microsoft Corporation. All rights reserved.

  Determining projects to restore...
/home/per/git/perlang/Perlang.ConsoleApp/Perlang.ConsoleApp.csproj : error NU3028: Package 'System.Memory 4.5.3' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain [/home/per/git/perlang/Perlang.sln]
/home/per/git/perlang/Perlang.ConsoleApp/Perlang.ConsoleApp.csproj : error NU3037: Package 'System.Memory 4.5.3' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired. [/home/per/git/perlang/Perlang.sln]
/home/per/git/perlang/Perlang.ConsoleApp/Perlang.ConsoleApp.csproj : error NU3028: Package 'System.Memory 4.5.3' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain [/home/per/git/perlang/Perlang.sln]
/home/per/git/perlang/Perlang.ConsoleApp/Perlang.ConsoleApp.csproj : error NU3028: Package 'System.CommandLine 2.0.0-beta1.20158.1' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain [/home/per/git/perlang/Perlang.sln]
/home/per/git/perlang/Perlang.ConsoleApp/Perlang.ConsoleApp.csproj : error NU3037: Package 'System.CommandLine 2.0.0-beta1.20158.1' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired. [/home/per/git/perlang/Perlang.sln]
/home/per/git/perlang/Perlang.ConsoleApp/Perlang.ConsoleApp.csproj : error NU3028: Package 'System.CommandLine 2.0.0-beta1.20158.1' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain [/home/per/git/perlang/Perlang.sln]
  Failed to restore /home/per/git/perlang/Perlang.ConsoleApp/Perlang.ConsoleApp.csproj (in 810 ms).
Attempting to cancel the build...
/usr/share/dotnet/sdk/5.0.102/NuGet.targets(131,5): error : Restore canceled! [/home/per/git/perlang/Perlang.sln]

Build FAILED.
```

`ca-certificates` package version is `20210119`. I had the old version installed before this morning, but even after upgrading to the fixed version (and even running `sudo update-ca-certificates` manually afterwards) I am still seeing this.

Here's a full listing of all my `dotnet`-related packages (or most of them). As can be seen, I have both .NET Core 3.1 and 5.0 installed side-by-side.

```
$ dpkg -l | grep dotnet
ii  dotnet-apphost-pack-3.1                                     3.1.11-1                                                  amd64        Microsoft.NETCore.App.Host 3.1.11
ii  dotnet-apphost-pack-5.0                                     5.0.2-1                                                   amd64        Microsoft.NETCore.App.Host 5.0.2
ii  dotnet-host                                                 5.0.2-1                                                   amd64        Microsoft .NET Host - 5.0.2
ii  dotnet-hostfxr-3.1                                          3.1.11-1                                                  amd64        Microsoft .NET Core Host FX Resolver - 3.1.11 3.1.11
ii  dotnet-hostfxr-5.0                                          5.0.2-1                                                   amd64        Microsoft .NET Host FX Resolver - 5.0.2 5.0.2
ii  dotnet-runtime-3.1                                          3.1.11-1                                                  amd64        Microsoft .NET Core Runtime - 3.1.11 Microsoft.NETCore.App 3.1.11
ii  dotnet-runtime-5.0                                          5.0.2-1                                                   amd64        Microsoft .NET Runtime - 5.0.2 Microsoft.NETCore.App 5.0.2
ii  dotnet-runtime-deps-3.1                                     3.1.11-1                                                  amd64        dotnet-runtime-deps-3.1 3.1.11
ii  dotnet-runtime-deps-5.0                                     5.0.2-1                                                   amd64        dotnet-runtime-deps-5.0 5.0.2
ii  dotnet-sdk-3.1                                              3.1.405-1                                                 amd64        Microsoft .NET Core SDK 3.1.405
ii  dotnet-sdk-5.0                                              5.0.102-1                                                 amd64        Microsoft .NET SDK 5.0.102
ii  dotnet-targeting-pack-3.1                                   3.1.0-1                                                   amd64        Microsoft.NETCore.App.Ref 3.1.0
ii  dotnet-targeting-pack-5.0                                   5.0.0-1                                                   amd64        Microsoft.NETCore.App.Ref 5.0.0
```

Let me know if I can aid in debugging this any further.

Addmitedly I am very new, but `dotnet new webapi` fails with cert errors outside Docker for me (running on my own computer running Ubuntu 21.04)

<details>
<summary>Relevant outputs</summary>

```
t430 :: dev/net/thawkapi 127 » dotnet restore
  Determining projects to restore...
/home/artur/dev/net/thawkapi/thawkapi.csproj : error NU3028: Package 'Microsoft.Extensions.ApiDescription.Server 3.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/home/artur/dev/net/thawkapi/thawkapi.csproj : error NU3037: Package 'Microsoft.Extensions.ApiDescription.Server 3.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired.
/home/artur/dev/net/thawkapi/thawkapi.csproj : error NU3028: Package 'Microsoft.Extensions.ApiDescription.Server 3.0.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/home/artur/dev/net/thawkapi/thawkapi.csproj : error NU3028: Package 'Microsoft.OpenApi 1.2.3' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/home/artur/dev/net/thawkapi/thawkapi.csproj : error NU3037: Package 'Microsoft.OpenApi 1.2.3' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired.
/home/artur/dev/net/thawkapi/thawkapi.csproj : error NU3028: Package 'Microsoft.OpenApi 1.2.3' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
  Failed to restore /home/artur/dev/net/thawkapi/thawkapi.csproj (in 668 ms).
t430 :: dev/net/thawkapi 1 »
```

```
t430 :: dev/net/thawkapi 130 » dotnet --version
5.0.103
```

`~/.nuget/NuGet/NuGet.Config`
```
﻿<?xml version="1.0" encoding="utf-8"?>
<configuration>
  <packageSources>
    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
  </packageSources>
  <config>
    <add key="signatureValidationMode" value="accept" />
    <add key="allowUntrustedRoot" value="accept" />
  </config>
</configuration>
```

Distro info:
```
Distributor ID:	Ubuntu
Description:	Ubuntu Hirsute Hippo (development branch)
Release:	21.04
Codename:	hirsute
```
</details>

> @kitingChris
> 
> > I tried to update the ca-certificates but already had the newest version
> > ```
> > $ sudo apt list --installed | grep ca-certificates
> > ca-certificates/now 20210119 all [installed,local]
> > ```
> 
> Did you try the suggestion I posted in [#10491 (comment)](https://github.com/NuGet/Home/issues/10491#issuecomment-778841003)? Maybe it would help. If not, please post the error message you're getting and we might be able to find a way forward.

I tried. And it is used as shown here:

```
$ ls -lsah /etc/ssl/certs | grep -E 'Microsoft|VeriSign'
   0 lrwxrwxrwx 1 root root   49 Jan 28 09:36 8d89cda1.0 -> Microsoft_ECC_Root_Certificate_Authority_2017.pem
   0 lrwxrwxrwx 1 root root   49 Jan 28 09:36 bf53fb88.0 -> Microsoft_RSA_Root_Certificate_Authority_2017.pem
   0 lrwxrwxrwx 1 root root   51 Feb 24 12:06 c01cdfa2.0 -> VeriSign_Universal_Root_Certification_Authority.pem
4,0K lrwxrwxrwx 1 root root   84 Jan 28 09:36 Microsoft_ECC_Root_Certificate_Authority_2017.pem -> /usr/share/ca-certificates/mozilla/Microsoft_ECC_Root_Certificate_Authority_2017.crt
4,0K lrwxrwxrwx 1 root root   84 Jan 28 09:36 Microsoft_RSA_Root_Certificate_Authority_2017.pem -> /usr/share/ca-certificates/mozilla/Microsoft_RSA_Root_Certificate_Authority_2017.crt
4,0K lrwxrwxrwx 1 root root   86 Feb 24 12:06 VeriSign_Universal_Root_Certification_Authority.pem -> /usr/share/ca-certificates/mozilla/VeriSign_Universal_Root_Certification_Authority.crt
```
The error message I still get:
```
$ dotnet restore

Welcome to .NET 5.0!
---------------------
SDK Version: 5.0.200

Telemetry
---------
The .NET tools collect usage data in order to help us improve your experience. It is collected by Microsoft and shared with the community. You can opt-out of telemetry by setting the DOTNET_CLI_TELEMETRY_OPTOUT environment variable to '1' or 'true' using your favorite shell.

Read more about .NET CLI Tools telemetry: https://aka.ms/dotnet-cli-telemetry

----------------
Installed an ASP.NET Core HTTPS development certificate.
To trust the certificate run 'dotnet dev-certs https --trust' (Windows and macOS only).
Learn about HTTPS: https://aka.ms/dotnet-https
----------------
Write your first app: https://aka.ms/dotnet-hello-world
Find out what's new: https://aka.ms/dotnet-whats-new
Explore documentation: https://aka.ms/dotnet-docs
Report issues and find source on GitHub: https://github.com/dotnet/core
Use 'dotnet --help' to see available commands or visit: https://aka.ms/dotnet-cli
--------------------------------------------------------------------------------------
  Determining projects to restore...
  Restored /home/chris/Workspace/MyProject.csproj (in 229 ms).
  Restored /home/chris/Workspace/MyProject/src/Database/MyProject.Model/MyProject.Model.csproj (in 229 ms).
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : warning NU3034: Package 'runtime.ubuntu.14.04-x64.runtime.native.System.Security.Cryptography.OpenSsl 4.3.0' from source 'https://pkgs.dev.azure.com/myOrg/_packaging/myOrg-feed/nuget/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : warning NU3034: Package 'runtime.native.System.Security.Cryptography.Apple 4.3.0' from source 'https://pkgs.dev.azure.com/myOrg/_packaging/myOrg-feed/nuget/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : warning NU3034: Package 'runtime.opensuse.42.1-x64.runtime.native.System.Security.Cryptography.OpenSsl 4.3.0' from source 'https://pkgs.dev.azure.com/myOrg/_packaging/myOrg-feed/nuget/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : warning NU3034: Package 'runtime.rhel.7-x64.runtime.native.System.Security.Cryptography.OpenSsl 4.3.0' from source 'https://pkgs.dev.azure.com/myOrg/_packaging/myOrg-feed/nuget/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : warning NU3034: Package 'runtime.osx.10.10-x64.runtime.native.System.Security.Cryptography.OpenSsl 4.3.0' from source 'https://pkgs.dev.azure.com/myOrg/_packaging/myOrg-feed/nuget/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : warning NU3034: Package 'runtime.fedora.24-x64.runtime.native.System.Security.Cryptography.OpenSsl 4.3.0' from source 'https://pkgs.dev.azure.com/myOrg/_packaging/myOrg-feed/nuget/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : warning NU3034: Package 'runtime.ubuntu.16.04-x64.runtime.native.System.Security.Cryptography.OpenSsl 4.3.0' from source 'https://pkgs.dev.azure.com/myOrg/_packaging/myOrg-feed/nuget/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Threading.Tasks.Extensions 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : warning NU3034: Package 'runtime.osx.10.10-x64.runtime.native.System.Security.Cryptography.Apple 4.3.0' from source 'https://pkgs.dev.azure.com/myOrg/_packaging/myOrg-feed/nuget/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : warning NU3034: Package 'runtime.ubuntu.16.10-x64.runtime.native.System.Security.Cryptography.OpenSsl 4.3.0' from source 'https://pkgs.dev.azure.com/myOrg/_packaging/myOrg-feed/nuget/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'runtime.opensuse.13.2-x64.runtime.native.System.Security.Cryptography.OpenSsl 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : warning NU3034: Package 'System.Reflection.Emit.Lightweight 4.3.0' from source 'https://pkgs.dev.azure.com/myOrg/_packaging/myOrg-feed/nuget/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'Microsoft.NETCore.Targets 1.1.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'runtime.fedora.23-x64.runtime.native.System.Security.Cryptography.OpenSsl 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : warning NU3034: Package 'System.Xml.XmlSerializer 4.3.0' from source 'https://pkgs.dev.azure.com/myOrg/_packaging/myOrg-feed/nuget/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Globalization.Extensions 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : warning NU3034: Package 'System.Numerics.Vectors 4.4.0' from source 'https://pkgs.dev.azure.com/myOrg/_packaging/myOrg-feed/nuget/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Runtime.Serialization.Formatters 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Diagnostics.Process 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Runtime.Serialization.Primitives 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'Newtonsoft.Json 10.0.1' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'NETStandard.Library 1.6.1' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Text.RegularExpressions 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Runtime.Numerics 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.ObjectModel 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Linq 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Threading 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Reflection 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Diagnostics.Debug 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Resources.ResourceManager 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Text.Encoding.Extensions 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.IO 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Collections 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Runtime.Extensions 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Globalization 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Threading.Tasks 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Text.Encoding 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Xml.ReaderWriter 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'Microsoft.CSharp 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Security.SecureString 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Linq.Expressions 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Threading.Thread 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Xml.XDocument 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'Microsoft.Win32.Primitives 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Dynamic.Runtime 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Security.Cryptography.X509Certificates 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.IO.FileSystem 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'runtime.native.System 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Reflection.Extensions 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Runtime.Handles 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Threading.ThreadPool 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Reflection.TypeExtensions 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.ComponentModel 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Security.Cryptography.Primitives 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Reflection.Emit.ILGeneration 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.IO.FileSystem.Primitives 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Runtime.InteropServices 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Diagnostics.Tools 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Collections.Specialized 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Reflection.Primitives 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Private.DataContractSerialization 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Reflection.Emit 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Globalization.Calendars 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Security.Cryptography.Cng 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Console 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Security.Cryptography.Encoding 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'runtime.native.System.Security.Cryptography.OpenSsl 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Collections.NonGeneric 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Runtime 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'runtime.native.System.Net.Http 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Diagnostics.Tracing 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Threading.Timer 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Security.Cryptography.Algorithms 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Security.Cryptography.OpenSsl 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.IO.Compression 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Net.Sockets 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Runtime.InteropServices.RuntimeInformation 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Collections.Concurrent 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Net.Primitives 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
/home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj : error NU3034: Package 'System.Security.Cryptography.Csp 4.3.0' from source 'https://api.nuget.org/v3/index.json': This package is signed but not by a trusted signer. [/home/chris/Workspace/MyProject/MyProject.sln]
  Failed to restore /home/chris/Workspace/MyProject/src/Database/MyProject.Database/MyProject.Database.csproj (in 6.44 sec).
```

Still an issue for me with 

```
$ dotnet --version
5.0.200
```

This is from the Chaotic AUR.

From the `dotnet-sdk` and `dotnet-runtime` packages from the Arch community repo, same thing:

```
$ dotnet --version
5.0.104
$ dotnet new webapi -n SomeAPI
Welcome to .NET 5.0!
---------------------
SDK Version: 5.0.104

----------------
Installed an ASP.NET Core HTTPS development certificate.
To trust the certificate run 'dotnet dev-certs https --trust' (Windows and macOS only).
Learn about HTTPS: https://aka.ms/dotnet-https
----------------
Write your first app: https://aka.ms/dotnet-hello-world
Find out what's new: https://aka.ms/dotnet-whats-new
Explore documentation: https://aka.ms/dotnet-docs
Report issues and find source on GitHub: https://github.com/dotnet/core
Use 'dotnet --help' to see available commands or visit: https://aka.ms/dotnet-cli
--------------------------------------------------------------------------------------
Getting ready...
The template "ASP.NET Core Web API" was created successfully.

Processing post-creation actions...
Running 'dotnet restore' on Commader/Commader.csproj...
  Determining projects to restore...
/home/my_user/Projects/Dotnet/C#/Commader/Commader.csproj : error NU3028: Package 'Microsoft.Extensions.ApiDescription.Server 3.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/home/my_user/Projects/Dotnet/C#/Commader/Commader.csproj : error NU3028: Package 'Microsoft.OpenApi 1.2.3' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/home/my_user/Projects/Dotnet/C#/Commader/Commader.csproj : error NU3037: Package 'Microsoft.Extensions.ApiDescription.Server 3.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired.
/home/my_user/Projects/Dotnet/C#/Commader/Commader.csproj : error NU3037: Package 'Microsoft.OpenApi 1.2.3' from source 'https://api.nuget.org/v3/index.json': The author primary signature validity period has expired.
/home/my_user/Projects/Dotnet/C#/Commader/Commader.csproj : error NU3028: Package 'Microsoft.Extensions.ApiDescription.Server 3.0.0' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
/home/my_user/Projects/Dotnet/C#/Commader/Commader.csproj : error NU3028: Package 'Microsoft.OpenApi 1.2.3' from source 'https://api.nuget.org/v3/index.json': The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain
  Failed to restore /home/my_user/Projects/Dotnet/C#/Commader/Commader.csproj (in 478 ms).
Restore failed.
Post action failed.
Description: Restore NuGet packages required by this project.
Manual instructions: Run 'dotnet restore'
```

Running the solution [here](https://devblogs.microsoft.com/nuget/microsoft-author-signing-certificate-update/#client-policy):

```
nuget trusted-signers Add -Name Microsoft2021 -CertificateFingerprint AA12DA22A49BCE7D5C1AE64CC1F3D892F150DA76140F210ABD2CBFFCA2C18A27 -FingerprintAlgorithm SHA256
```

Changes nothing.

Uninstalling both the Arch community repo, removing the `~/.nuget/` folder, and installing the Arch packages again also doesn't fix anything. Same with installing the packages from the Chaoctic AUR.

--- a/extra-devel/aspnetcore-runtime-5.0/spec
+++ b/extra-devel/aspnetcore-runtime-5.0/spec
@@ -1,11 +1,9 @@
-VER=5.0.3
+VER=5.0.11
 
 SRCS__AMD64="https://dotnetcli.azureedge.net/dotnet/aspnetcore/Runtime/$VER/aspnetcore-runtime-$VER-linux-x64.tar.gz"
-CHKSUMS__AMD64="sha512::188618b07cd5c97a34c77efccc7c4eba10a52681592ef711a005c82d243e601891418e0dcc27a22aae7dd6ae33d35e0bb7aa9b9fc022746c6c2414bd25cdb110"
+CHKSUMS__AMD64="sha512::2843b60a599fcb269bf700ea6233653264529d6522808c1b6ae0dc119e52e2c3203f3050dfef441dcb44130031a9ae03adbf25f1a51605ddc6f696b91433a667"
 SRCS__ARM64="https://dotnetcli.azureedge.net/dotnet/aspnetcore/Runtime/$VER/aspnetcore-runtime-$VER-linux-arm64.tar.gz"
-CHKSUMS__ARM64="sha512::4eaf1b0120479102b342f1f3a8ad8f40b7326e3657d2b7359c09fd1951c5169ca02d1ead4d4d01f6e594adbe0cb21f135f4c61ff90613219596f6cc222161717"
+CHKSUMS__ARM64="sha512::70844c1c1ca12dbdef3c459ce9a27d5ac91c893891473efb7b2184596210008674078c9d9833d6a2d5114b680a182a3e83d135b3d175bb373b439fcf22f4fc02"
 
 SUBDIR=.
-REL=1
-CHKSUM="sha256::c220edde6058fd749fb22cfe7bb432378659c2941f60f0a9865998767d63e95d"
 CHKUPDATE="html::url=https://dotnetcli.azureedge.net/dotnet/release-metadata/releases-index.json;pattern=\"latest-runtime\": \"(5\..+?)\""

--- a/extra-devel/dotnet-sdk-5.0.4xx/autobuild/build
+++ b/extra-devel/dotnet-sdk-5.0.4xx/autobuild/build
@@ -1,0 +1,3 @@
+abinfo "Deploying files ..."
+mkdir -pv "$PKGDIR"/usr/lib/dotnet
+cp -rv "$SRCDIR"/sdk "$PKGDIR"/usr/lib/dotnet

--- a/extra-devel/dotnet-sdk-5.0.4xx/autobuild/defines
+++ b/extra-devel/dotnet-sdk-5.0.4xx/autobuild/defines
@@ -1,0 +1,11 @@
+PKGNAME=dotnet-sdk-5.0.4xx
+PKGSEC=devel
+PKGDES=".NET Core is a cross-platform .NET implementation for websites, servers, and console apps on Windows, Linux, and macOS"
+PKGDEP="dotnet-runtime-5.0 gcc-runtime glibc"
+PKGDEP__AMD64="$PKGDEP aspnetcore-runtime-5.0"
+PKGDEP__ARMEL="$PKGDEP aspnetcore-runtime-5.0"
+
+FAIL_ARCH="!(amd64|arm64)"
+NOSTATIC=0
+
+ABSPLITDBG=0

--- a/extra-devel/dotnet-sdk-5.0.4xx/spec
+++ b/extra-devel/dotnet-sdk-5.0.4xx/spec
@@ -1,0 +1,9 @@
+VER=5.0.402
+
+SRCS__AMD64="https://dotnetcli.azureedge.net/dotnet/Sdk/$VER/dotnet-sdk-$VER-linux-x64.tar.gz"
+CHKSUMS__AMD64="sha512::6b2937ad1f026fd91d0c8e6101b0422a8d19ead022055021b55a722c34dcc3697b592592eacc6def8748981bd996bc2a511d7c3f05b0ae431c00ede0376deacc"
+SRCS__ARM64="https://dotnetcli.azureedge.net/dotnet/Sdk/$VER/dotnet-sdk-$VER-linux-arm64.tar.gz"
+CHKSUMS__ARM64="sha512::9681da343f72998590541e0ee454a3d68aa4caff761ecc727c837fdbe8645c11487d8e8c4f6f9d3cda35373cee046b4857289a2307aa4929878633d13443e5bf"
+
+SUBDIR=.
+CHKUPDATE="html::url=https://dotnetcli.azureedge.net/dotnet/release-metadata/releases-index.json;pattern=\"latest-sdk\": \"(5\..+?)\""

--- a/extra-devel/dotnet-sdk-5.0/autobuild/defines
+++ b/extra-devel/dotnet-sdk-5.0/autobuild/defines
@@ -1,7 +1,7 @@
 PKGNAME=dotnet-sdk-5.0
 PKGSEC=devel
 PKGDES="Transitional package for most recent .NET Core platform SDK 5.0"
-PKGDEP="dotnet-sdk-5.0.1xx"
+PKGDEP="dotnet-sdk-5.0.4xx"
 
 FAIL_ARCH="!(arm64|amd64)"
 ABSPLITDBG=0

--- a/extra-devel/dotnet-sdk-5.0/spec
+++ b/extra-devel/dotnet-sdk-5.0/spec
@@ -1,5 +1,4 @@
-VER=5.0.103
+VER=5.0.402
 DUMMYSRC=1
-REL=1
 CHKSUMS="SKIP"
 CHKUPDATE="html::url=https://dotnetcli.azureedge.net/dotnet/release-metadata/releases-index.json;pattern=\"latest-sdk\": \"(5\..+?)\""

--- a/extra-devel/dotnet-sdk-5/spec
+++ b/extra-devel/dotnet-sdk-5/spec
@@ -1,5 +1,4 @@
-VER=5.0.103
+VER=5.0.402
 DUMMYSRC=1
-REL=1
 CHKSUMS="SKIP"
 CHKUPDATE="html::url=https://dotnetcli.azureedge.net/dotnet/release-metadata/releases-index.json;pattern=\"latest-sdk\": \"(5\..+?)\""

--- a/extra-libs/dotnet-host-fxr/spec
+++ b/extra-libs/dotnet-host-fxr/spec
@@ -1,9 +1,9 @@
-VER=5.0.3
+VER=5.0.11
 
 SRCS__AMD64="https://dotnetcli.azureedge.net/dotnet/Runtime/$VER/dotnet-runtime-$VER-linux-x64.tar.gz"
-CHKSUMS__AMD64="sha512::263dbe260123c3d6d706ed8b5f4d510d9384216422e9af0d293df87ed98e84e1e0ffbf0c7dd543c40c5ccc95bd7cd006c8bbbe9f1cd1f060b1eaa2f7a60fea74"
+CHKSUMS__AMD64="sha512::4eef891385288e5ab35beea79e2e6ca13615afe2e90da7a7a14417119bdbeb42980dda5f1cbe5b6d8e05a728d0db4256abb54cd00a63aa621a70090bddbe9f69"
 SRCS__ARM64="https://dotnetcli.azureedge.net/dotnet/Runtime/$VER/dotnet-runtime-$VER-linux-arm64.tar.gz"
-CHKSUMS__ARM64="sha512::f4d176b48714121040470a25f76a1b3554edb358953c1bfb39bd111cf0dcc85cb8a5ebcd617efd207b3cfaef0a2d242f94e9f018a186828a750c5c0dc9bd7da5"
+CHKSUMS__ARM64="sha512::b296a79f655ad22db405f24c1549b6cd610c57dae917c89b91ddcb4b053a7613952261cca7d7c79e6fda72026fd4bbbcf74751cb6529a55504a67223cabc0451"
 
 SUBDIR=.
 CHKUPDATE="html::url=https://dotnetcli.azureedge.net/dotnet/release-metadata/releases-index.json;pattern=\"latest-release\": \"(5\..+?)\""

--- a/extra-libs/dotnet-host/spec
+++ b/extra-libs/dotnet-host/spec
@@ -1,9 +1,9 @@
-VER=5.0.3
+VER=5.0.11
 
 SRCS__AMD64="https://dotnetcli.azureedge.net/dotnet/Runtime/$VER/dotnet-runtime-$VER-linux-x64.tar.gz"
-CHKSUMS__AMD64="sha512::263dbe260123c3d6d706ed8b5f4d510d9384216422e9af0d293df87ed98e84e1e0ffbf0c7dd543c40c5ccc95bd7cd006c8bbbe9f1cd1f060b1eaa2f7a60fea74"
+CHKSUMS__AMD64="sha512::4eef891385288e5ab35beea79e2e6ca13615afe2e90da7a7a14417119bdbeb42980dda5f1cbe5b6d8e05a728d0db4256abb54cd00a63aa621a70090bddbe9f69"
 SRCS__ARM64="https://dotnetcli.azureedge.net/dotnet/Runtime/$VER/dotnet-runtime-$VER-linux-arm64.tar.gz"
-CHKSUMS__ARM64="sha512::f4d176b48714121040470a25f76a1b3554edb358953c1bfb39bd111cf0dcc85cb8a5ebcd617efd207b3cfaef0a2d242f94e9f018a186828a750c5c0dc9bd7da5"
+CHKSUMS__ARM64="sha512::b296a79f655ad22db405f24c1549b6cd610c57dae917c89b91ddcb4b053a7613952261cca7d7c79e6fda72026fd4bbbcf74751cb6529a55504a67223cabc0451"
 
 SUBDIR=.
 CHKUPDATE="html::url=https://dotnetcli.azureedge.net/dotnet/release-metadata/releases-index.json;pattern=\"latest-release\": \"(5\..+?)\""

--- a/extra-libs/dotnet-runtime-5.0/spec
+++ b/extra-libs/dotnet-runtime-5.0/spec
@@ -1,9 +1,9 @@
-VER=5.0.3
+VER=5.0.11
 
 SRCS__AMD64="https://dotnetcli.azureedge.net/dotnet/Runtime/$VER/dotnet-runtime-$VER-linux-x64.tar.gz"
-CHKSUMS__AMD64="sha512::263dbe260123c3d6d706ed8b5f4d510d9384216422e9af0d293df87ed98e84e1e0ffbf0c7dd543c40c5ccc95bd7cd006c8bbbe9f1cd1f060b1eaa2f7a60fea74"
+CHKSUMS__AMD64="sha512::4eef891385288e5ab35beea79e2e6ca13615afe2e90da7a7a14417119bdbeb42980dda5f1cbe5b6d8e05a728d0db4256abb54cd00a63aa621a70090bddbe9f69"
 SRCS__ARM64="https://dotnetcli.azureedge.net/dotnet/Runtime/$VER/dotnet-runtime-$VER-linux-arm64.tar.gz"
-CHKSUMS__ARM64="sha512::f4d176b48714121040470a25f76a1b3554edb358953c1bfb39bd111cf0dcc85cb8a5ebcd617efd207b3cfaef0a2d242f94e9f018a186828a750c5c0dc9bd7da5"
+CHKSUMS__ARM64="sha512::b296a79f655ad22db405f24c1549b6cd610c57dae917c89b91ddcb4b053a7613952261cca7d7c79e6fda72026fd4bbbcf74751cb6529a55504a67223cabc0451"
 
 SUBDIR=.
-CHKUPDATE="html::url=https://dotnetcli.azureedge.net/dotnet/release-metadata/releases-index.json;pattern=\"latest-runtime\": \"(5\..+?)\""
+CHKUPDATE="html::url=https://dotnetcli.azureedge.net/dotnet/release-metadata/releases-index.json;pattern=\"latest-release\": \"(5\..+?)\""

